src/screens/AuthScreen/Util.ts

```diff
@@ -336,35 +354,56 @@ export const formatTimeAgo = (timestamp) => {
 };
 
 export const fetchTaskDetails = async (
-  token: string,
   taskId: String,
+  token: string,
 ): Promise<any> => {
+  console.log(
+    'token and taskid',
+    urls.GET_ALL_TASK + taskId + '/details',
+    taskId,
+  );
   try {
     const response = await axios.get(urls.GET_ALL_TASK + taskId + '/details', {
       headers: {
         'Content-type': 'application/json',
         cookie: `rds-session=${token}`,
       },
     });
+    console.log(response.status);
```

Comment:
Approving it for now but you have a lot of console logs in the project.
Please create a PR removing all of them.